cozy/common.py

```diff
@@ -323,8 +337,17 @@ def nested_dict(n, t):
         return t()
     return OrderedDefaultDict(lambda: nested_dict(n-1, t))
 
+## TODO: Rename this varable; for example, _fresh_names_counter.
 _i = Value(ctypes.c_uint64, 0)
+
+## TODO: This routine is never used, except by fresh_name which passes n=1.
+## I would remove this routine.
```

Comment:
At some point I had thought to have workers "pre-allocate" blocks of names and thereby reduce interprocess locking. It never turned out to be a bottleneck, so I suppose we can do away with the feature. Still, I will remember this bulk API fondly.